grafana-plugin/src/models/alertgroup/alertgroup.helpers.ts

```diff
@@ -33,6 +35,26 @@ export class AlertGroupHelper {
     return (await onCallApi().POST('/alertgroups/bulk_action/', { params: {}, body: data })).data;
   }
 
+  @AutoLoadingState(ActionKey.INCIDENTS_BULK_UPDATE)
+  static async updateBulkActionAndRefresh(
+    data: ApiSchemas['AlertGroupBulkActionRequest'],
+    alertGroupStore: AlertGroupStore,
+    onFinally?: () => void
+  ) {
+    try {
+      alertGroupStore.setLiveUpdatesPaused(true);
+
+      await AlertGroupHelper.bulkAction(data);
+
+      // pull new data
+      await alertGroupStore.fetchAlertGroups();
+    } finally {
+      alertGroupStore.setLiveUpdatesPaused(false);
+
+      onFinally?.();
+    }
```

Comment:
Why do we need try {} finally {}? Afaik it doesn't make much sense if we omit `catch`, we could just call those functions in the same order without try{}finally{} or I'm missing sth?

Also this way I think we don't handle errors if they appear in try {} block, right?

Reply:
My idea was to make sure we're calling `setLiveUpdatesPaused` and the passed handler even if the request fails. I guess we could add a catch clause and show notification?

Reply:
In this we can add catch clause and at least rethrow the error so it's handled by our error boundaries. Without it the error will be completely swallowed